gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/test/java/io/gravitee/am/gateway/handler/scim/service/UserServiceTest.java

```diff
@@ -730,4 +730,37 @@ public void shouldDeleteUser_noExternalProvider() {
         verify(identityProviderManager, times(1)).getUserProvider(anyString());
         verify(userProvider, never()).delete(anyString());
     }
+
+    @Test
+    public void shouldCreateUser_with_lastPasswordReset() {
+
+        Date lastPasswordResetDate = new Date();
+
+        GraviteeUser newUser = mock(GraviteeUser.class);
+        when(newUser.getSource()).thenReturn("unknown-idp");
+        when(newUser.getUserName()).thenReturn("username");
+        when(newUser.getPassword()).thenReturn(null);
+        Map<String, Object> ai = new HashMap<>();
+        ai.put("lastPasswordReset", lastPasswordResetDate);
```

Comment:
question: Should we use Instant instead of Date and use the Instant.toString method to have the right format (don't know if the toString is using the format used in the SCIM spec) ?